include/sound/sof/ext_manifest.h

```diff
@@ -71,4 +73,11 @@ struct ext_man_fw_version {
 	uint32_t flags;
 } __packed;
 
+/* extended data memory windows for IPC, trace and debug */
+struct ext_man_window {
+	struct ext_man_elem_header hdr;
+	/* use sof_ipc_window struct because of backward compatibility issue */
```

Comment:
could you explain what "compatibility issue" you mean here? Do you mean *for* compatibility and code re-use?

Reply:
changed